mysql-test/suite/rocksdb/r/collation.result

```diff
@@ -117,12 +60,11 @@ CREATE TABLE c (id INT PRIMARY KEY, value varchar(50), index(value)) engine=rock
 ERROR HY000: Unsupported collation on string indexed column test.c.value Use binary collation (binary, latin1_bin, utf8_bin).
 DROP TABLE a, b;
 SET GLOBAL rocksdb_strict_collation_exceptions="abc\\";
- Invalid pattern in strict_collation_exceptions: abc\
 CREATE TABLE abc (id INT PRIMARY KEY, value varchar(50), index(value)) engine=rocksdb charset utf8;
 ERROR HY000: Unsupported collation on string indexed column test.abc.value Use binary collation (binary, latin1_bin, utf8_bin).
 SET GLOBAL rocksdb_strict_collation_exceptions="abc";
 CREATE TABLE abc (id INT PRIMARY KEY, value varchar(50), index(value)) engine=rocksdb charset utf8;
-CREATE TABLE abcd (id INT PRIMARY KEY, value varchar(50), index(value)) engine=rocksdb charset utf8;
-ERROR HY000: Unsupported collation on string indexed column test.abcd.value Use binary collation (binary, latin1_bin, utf8_bin).
+CREATE TABLE bcd (id INT PRIMARY KEY, value varchar(50), index(value)) engine=rocksdb charset utf8;
```

Comment:
Actually, this is slightly incorrect. The original test is incorrect. A proper regex of "abc" should match "abcd" as "abc" is within "abcd".

Within MySQL REGEX:

> mysql> SELECT 'abcd' REGEXP 'abc';
> +---------------------+
> | 'abcd' REGEXP 'abc' |
> +---------------------+
> |                   1 |
> +---------------------+
> 1 row in set (0.00 sec)

std::regex should make this same match based on the documented syntax http://en.cppreference.com/w/cpp/regex/ecmascript but it does not.